benches/bench_table.rs

```diff
@@ -0,0 +1,124 @@
+mod common;
+
+use agatedb::{Table, TableBuilder, TableOptions, Value};
+use bytes::Bytes;
+use common::rand_value;
+use criterion::{criterion_group, criterion_main, Criterion};
+use rand::Rng;
+use tempdir::TempDir;
+
+fn bench_table_builder(c: &mut Criterion) {
+    c.bench_function("table builder", |b| {
+        let mut key_list = vec![];
+
+        const KEY_COUNT: usize = 1300000; // about 64MB
+
+        for i in 0..KEY_COUNT {
+            let k = Bytes::from(format!("{:032}", i));
+            key_list.push(k);
+        }
+
+        let vs = Value::new(Bytes::from(rand_value()));
+
+        let opt = TableOptions {
+            block_size: 4 * 1024,
+            bloom_false_positive: 0.01,
+            table_size: 5 << 20,
+        };
+
+        b.iter(|| {
+            let mut builder = TableBuilder::new(opt.clone());
+            for j in 0..KEY_COUNT {
+                builder.add(&key_list[j], vs.clone(), 0);
+            }
+            builder.finish()
+        });
+    });
+}
+
+fn get_table_for_benchmark(count: usize) -> Table {
+    let tmp_dir = TempDir::new("agatedb").unwrap();
```

Comment:
This can leave dirty file in temporary directory on Windows.

Reply:
What about passing table in a closure, like this changes that will land in following PRs?

https://github.com/tikv/agatedb/blob/070a61113d82d2e111a86f2a99be11b8bc30691c/src/db.rs#L320

Reply:
As this issue also exists in tests, and using schemes like `with_table_test` requires large refactor, I would like to fix this in a separate PR.